pkg/storer/reserve.go

```diff
@@ -135,6 +135,9 @@ func (db *DB) reserveSizeWithinRadiusWorker(ctx context.Context) {
 			if exists, _ := db.batchstore.Exists(ci.BatchID); !exists {
 				missing++
 				db.logger.Debug("reserve size worker, item with invalid batch id", "batch_id", hex.EncodeToString(ci.BatchID), "chunk_address", ci.ChunkAddress)
+				if err := db.EvictBatch(ctx, ci.BatchID); err != nil {
+					db.logger.Error(err, "reserve size worker, batch eviction", "batch_id", hex.EncodeToString(ci.BatchID), "chunk_address", ci.ChunkAddress)
```

Comment:
Is this error important to the node operator? If the node can continue its normal operation use debug level.